caesar/caesar.js

```diff
@@ -0,0 +1,18 @@
+const caesar = (text, key) => text.split('').map(ch => shift(ch, key)).join('');
```

Comment:
You are using function declarations on all the other files. Try to stick with one style, either function declarations or function expressions, to be consistent.